lib/test/t_LeastSquaresExpansion_std.cxx

```diff
@@ -0,0 +1,158 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of the LeastSquaresExpansion class
+ *
+ *  Copyright 2005-2022 Airbus-EDF-IMACS-ONERA-Phimeca
```

Comment:
2023

Reply:
This is still not fixed, from my point of view.

Reply:
This is still not fixed, from my point of view.